pkg/kvstore/etcd.go

```diff
@@ -580,7 +580,7 @@ reList:
 					}
 
 					// Retrieve latest revision
-					lastRev = 0
+					// lastRev = 0
```

Comment:
It looks to me like this should be reused at the start of the `reList` loop, so it's not really ineffassign. Maybe we can find a way to teach ineffassign about this?

Reply:
I think it will be hard to teach ineffassign about this (it just doesn't have a support of skipping errors). But I have an another idea - we can wrap `ineffassign` call in `contrib/scripts/check-ineffassign.sh` and do there something like:

```
ineffassign $TOPDIR | grep -v "# ineffassign: ignore"
````

Because ineffassign output shows the whole line.